### ISSUE TITLE
asciidoctor not in standard image

### ISSUE THREAD

**Issue by rdmueller** [gitpod-io/workspace-images#265]
### Describe the bug
It seems that the gitpod IDE supports rendering of AsciiDoc files but asciidoctor (the tool to render the file) is not part of the standard image

### Steps to reproduce

start https://github.com/JohannesDienstDBSystel/spock-mutation-testing in gitpod. You will see the `README.adoc` as preview showing an error message.

### Expected behavior

I expect to see the `.adoc file rendred properly.

### Additional information

I guess it would be enough to add 

    gem install asciidoctor

To the base docker image. See https://asciidoctor.org/docs/install-toolchain/ for more details on how to install asciidoctor

### Example repository

https://github.com/JohannesDienstDBSystel/spock-mutation-testing

**Comment by sr229** [comment]
> It seems that asciidoctor has been removed with this PR: https://github.com/gitpod-io/workspace-images/pull/152/files
> If asciidoctor makes the image size too big, would it make sense to select the docker image in a smart way? Like detecting `.adoc` files and then switching to the asciidoctor image?

We'd definitely want a workspace image that is seperate if ASCIIDoc is too large for the base image.

**Comment by ret394** [comment]
(snipped)

TLDR - Not everyone needs asciidoctor.

I'm not a doctor. But I think bloating body of X could use some mitigations.

----

https://github.com/gitpod-io/workspace-images/issues/265#issuecomment-690826738

>We'd definitely want a workspace image that is seperate if ASCIIDoc is too large for the base image.

What size is it currently? the threshold for it?

---

I'll create yet another Github issue today, the damage cannot be undone.

**Comment by rdmueller** [comment]
@ret394 sorry, but I do not fully understand your comments.

yes, not everybody needs asciidoctor, but it once was part of the image and removing it breaks some repos.

I am a bit surprised that we really talk about the image size, because I guess chaching will be quite helpful to mitigate this and the base image already contains lot of stuff which not every body needs. But it is ok for me that the size will be limited.
I will create and reference my own asciidoc image, based on the gitpod base image, next time I need it.

why do you reference #2178?

**Comment by ret394** [comment]
@rdmueller 

> I am a bit surprised that we really talk about the image size

I want to be nice, but if you haven't got the juicy details of conducting the equivalent of a bypass heart surgery in a living human in software engineering. I'll have to ignore everyone that starts off sentences with banal statements. 


2178? I don't know, I'm just built that way.

---

to other geniuses

Hint - Look at the code! And my previous comments. If ret394 ignores you.
Hint - The hide comment due to reason x is also used by me to min it fast. I don't care about the ordinary use case of hiding issue for your stated reasons, M trumps your Y, your Z and your X.

**Comment by kylos101** [comment]
Hey there :wave: , we are closing this issue due to inactivity. Please feel free to comment or reopen if it is still an issue? :pray:
